src/xdebugger.cpp

```diff
@@ -273,6 +286,10 @@ namespace xpyt
             py::gil_scoped_acquire acquire;
             py::module xeus_python_shell = py::module::import("xeus_python_shell.debugger");
             m_pydebugger = xeus_python_shell.attr("XDebugger")();
+
+            std::string expression = "debugpy.__version__";
+            py::object version = eval(py::str(expression));
+            m_debugpy_version = version.cast<std::string>();
```

Comment:
A good idea here would be to replace the `.` char in `m_debug_py` with spaces, to ease future comparison:

```cpp
auto debugpy_version = version.cast<std::string>();
std::replace(debugpy_version.begin(), debugpy_version.end(), '.' , ' ');
```

You can then initialize the boolean member by calling a comparison function with debugpy_version and the minimal version of debugpy in the same format (i.e. "1 6 5");

Reply:
I don't see the difference if we replace '.' with ' '.
Can it simplify the comparison ?

Reply:
Yes because we can then use the istream_iterator as a tokenizer (it skips the spaces in a string by default, see the implementation in the comment below).